redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -139,19 +151,15 @@ public function log($success, $message)
     }
 
     /**
-     * @template T of rex_cronjob
-     *
-     * @return class-string<T>[]
+     * @return list<class-string<rex_cronjob>>
```

Comment:
Die neue phpstan-version meckert hier, da das template in keinem parameter verwendet wird: https://github.com/redaxo/redaxo/pull/4417/files
Das Template scheint mir hier auch unnötig zu sein.